app/assets/javascripts/components/campaign/campaign_ores_plot.jsx

```diff
@@ -65,7 +66,14 @@ const CampaignOresPlot = createReactClass({
       }
       return <div>No Structural Completeness data available</div>;
     }
-    return (<button className="button small" onClick={this.show}>Change in Structural Completeness</button>);
+    return (
+      <div className="container">
+        <section className="overview container">
+          <CampaignStats campaign={this.props.campaign} />
```

Comment:
This should be rendered all the time, not just before the `show` state is activated. I think it makes more sense to put this into campaign.jsx and have it appear on the Alerts tab as well, instead of rendering it from campaign_ores_plot.jsx.